HW4/sort.cpp

```diff
@@ -0,0 +1,178 @@
+#include "Functions.h"
+#include "sort.h"
+
+auto swap = [](auto &elOne, auto &elTwo) {
+  auto buf{elOne};
+  elOne = elTwo;
+  elTwo = buf;
+};
+
+void buble_sort(int *begin, int *end, Pred pred) {
+  if (begin == nullptr || end == nullptr)
+    return;
+  int i = 1;
```

Comment:
Застосовуйте єдиний стиль ініціалізації змінних
```suggestion
  int i  {1};
```
Краще якщо це буде ініверсальний